File: eval/baselines/settings/flutter/settings_screen.dart
// BASELINE — intentionally-naive "no-skill" reference for the eval harness (do NOT
// copy). What an assistant typically emits WITHOUT the Mobile UI/UX Pro Max skill:
// hardcoded colors, off-grid spacing, physical left/right insets, a destructive
// action inline with the rest (not isolated), sub-legible captions, and only the
// happy path with none of the required UI states. Graded against
// examples/settings/flutter/.
import 'package:flutter/material.dart';

class SettingsScreen extends StatefulWidget {
  const SettingsScreen({super.key});
  @override
  State<SettingsScreen> createState() => _SettingsScreenState();
}

class _SettingsScreenState extends State<SettingsScreen> {
  bool notifications = true;
  bool darkTheme = false;

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      backgroundColor: const Color(0xFFFFFFFF),
      appBar: AppBar(title: const Text('Settings')),
      body: ListView(
        padding: const EdgeInsets.only(left: 18, right: 18, top: 10),
        children: [
          SwitchListTile(
            title: const Text('Notifications'),
            value: notifications,
            onChanged: (v) => setState(() => notifications = v),
          ),
          SwitchListTile(
            title: const Text('Dark theme'),
            value: darkTheme,
            onChanged: (v) => setState(() => darkTheme = v),
          ),
          const SizedBox(height: 15),
          const Text('Signed in as user@example.com', style: TextStyle(fontSize: 10)),
          TextButton(
            onPressed: () {},
            child: const Text('Delete account', style: TextStyle(color: Color(0xFFDC2626))),
          ),
        ],
      ),
    );
  }
}

File: eval/baselines/settings/swiftui/SettingsScreen.swift
// BASELINE — intentionally-naive "no-skill" reference for the eval harness (do NOT
// copy). What an assistant typically emits WITHOUT the Mobile UI/UX Pro Max skill:
// hardcoded hex colors, off-grid spacing, a fixed-height caption row, a destructive
// action inline with the rest (not isolated), and only the happy path with none of
// the required UI states. Graded against examples/settings/swiftui/. (Assumes a
// Color(hex:) extension.)
import SwiftUI

struct SettingsScreen: View {
    @State private var notifications = true
    @State private var darkTheme = false

    var body: some View {
        VStack(alignment: .leading) {
            Text("Settings").foregroundColor(Color(hex: "#111827"))
            Toggle("Notifications", isOn: $notifications).padding(15)
            Toggle("Dark theme", isOn: $darkTheme).padding(15)
            Text("Signed in as user@example.com")
                .foregroundColor(Color(hex: "#6B7280"))
                .frame(height: 14)
            Button(action: {}) { Text("Delete account").foregroundColor(Color(hex: "#DC2626")) }
        }
        .padding(18)
    }
}
